.github/CODEOWNERS

```
@@ -0,0 +1,19 @@
+# default owners for anything not listed below, such as README and CI workflows
+* @krivard @capnrefsmmat @ryantibs
+
+# covidcast R package and data generation
+/R-packages/covidcast/ @capnrefsmmat @ryantibs @sgsmob
+/R-packages/data-raw/ @capnrefsmmat @ryantibs @sgsmob
+
+# evalcast R package
+/R-packages/evalcast/  @ryantibs @jacobbien
+
+# covidcast Python package
+/Python-packages/covidcast-py/ @chinandrew @capnrefsmmat
+
+# notebooks
+/R-notebooks/  @ryantibs @benjaminysmith
```

Collin's agreed to be the statistical reviewer for dashboards.

```suggestion
/R-notebooks/  @benjaminysmith @capolitsch @krivard 
```